lib/msf/core/handler/reverse_http/stageless.rb

```diff
@@ -26,7 +26,7 @@ def initialize_stageless
     ], self.class)
   end
 
-  def generate_stageless(&block)
+  def generate_stageless(ssl, &block)
```

Comment:
that's what I was alluding to with the passing an option hash, or even a cool keyword argument :)